benchmarks/basic/gocql/source/config.go

```diff
@@ -0,0 +1,87 @@
+package main
+
+import "flag"
+
+type Workload int
+
+const (
+	Inserts Workload = iota
+	Selects
+	Mixed
+)
+
+type Config struct {
+	nodeAddress string
+	workload    Workload
+	tasks       int64
+	concurency  int64
+	batchSize   int64
+	noPrepare   bool
+}
+
+func readConfig() Config {
+	config := Config{}
+
+	flag.StringVar(
+		&config.nodeAddress,
+		"address",
+		"scylla:9042",
+		"Address of database node to connect to",
+	)
+
+	workload := flag.String(
+		"workload",
+		"mixed",
+		"Type of work to perform (inserts, selects, mixed)",
+	)
+
+	flag.Int64Var(
+		&config.tasks,
+		"tasks",
+		1_000_000,
+		"Total number of tasks (requests) to perform the during benchmark. In case of mixed workload there will be tasks/2 inserts and tasks/2 selects",
```

Comment:
In case of `Mixed` other benchmarks do `tasks` inserts and `tasks` selects.
Maybe the README is not clear enough on this

It looks like this benchmark does the correct thing, but the message is wrong

Reply:
This was a copy paste from [rust benchmark](https://github.com/havaker/rust-driver-benchmarks/blob/d6be0b76dc1796e9ac12cc86d6b6c2cf17c3cffb/benchmarks/basic/rust/source/src/config.rs#L40)

Reply:
Oops I will fix that 
should be just `tasks` without the `/2`

Reply:
is `"Total number of tasks (requests) to perform the during benchmark. In case of mixed workload there will be tasks inserts and tasks selects"` ok?

Reply:
Its perfect